File: 158A - Next Round/NextRound.java
import java.util.Scanner;

public class NextRound {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String in = sc.nextLine();
        int count = 0;
        int n = Integer.parseInt(in.split(" ")[0]);
        int k = Integer.parseInt(in.split(" ")[1]);

        int a[] = new int[n];

        in = sc.nextLine();
        for (int i = 0; i < n; i++)
            a[i] = Integer.parseInt(in.split(" ")[i]);

        for (int i = 0; i < n; i++)
            if (a[i] >= a[k - 1] && a[i] != 0)
                count++;

        System.out.println(count);

        sc.close();
    }
}

File: 158A - Next Round/NextRound.cs
using System;

class NextRound
{
    static void Main()
    {
        string input = Console.ReadLine();
        int count = 0;
        int n = int.Parse(input.Split(' ')[0]);
        int k = int.Parse(input.Split(' ')[1]);

        int[] a = new int[n];
        input = Console.ReadLine();

        for (int i = 0; i < n; i++)
        {
            a[i] = int.Parse(input.Split(' ')[i]);
        }

        for (int i = 0; i < n; i++)
            if (a[i] >= a[k - 1] && a[i] != 0)
                count++;


        Console.WriteLine(count);
    }
}